app/orcaflex/api.py

```diff
@@ -217,4 +221,14 @@ def stop_jobs(job_ids):
     Job.stop(job_ids)
     
     worker_queue.clear()
-    
\ No newline at end of file
+    
+with app.app_context():
+    json_data = filing.get_all_jobs_json()
+    print(json_data)
```

Comment:
Mr print can be banished

Although maybe a log of how many jobs were restarted would be handy - there are examples of logging somewhere in this file.